cobbler.spec

```diff
@@ -54,7 +54,7 @@
 %define apache_group www
 
 %define apache_dir /srv/www
-%define apache_webconfigdir /etc/apache2/vhosts.d
+%define apache_webconfigdir /etc/apache2/conf.d
```

Comment:
This change makes no sense.

Reply:
Reverted.

Reply:
@Conan-Kudo Why do you think that this makes no sense? The previous version also installed to `conf.d` instead of `vhosts.d`.

Reply:
I guess we can resolve this.